src/widgets/avformatproducerwidget.cpp

```diff
@@ -369,6 +369,33 @@ void AvformatProducerWidget::onFrameDecoded()
                     ui->rangeComboBox->setEnabled(false);
                 }
                 ui->videoTableWidget->setItem(3, 1, new QTableWidgetItem(pix_fmt));
+                key = QString("meta.media.%1.codec.colorspace").arg(i);
+                int colorspace = m_producer->get_int(key.toLatin1().constData());
+                QString csString = tr("unknown");
+                switch (colorspace) {
+                    case 240: csString = "SMPTE ST240"; break;
+                    case 601: csString = "ITU-R BT.601"; break;
+                    case 709: csString = "ITU-R BT.709"; break;
+                    case 2020: csString = "ITU-R BT.2020"; break;
```

Comment:
`case 2020` will have to change per the latest MLT branch.

Reply:
Done.